library/src/main/scala/scala/scalajs/runtime/WasPublicBeforeTyper.scala

```diff
@@ -0,0 +1,5 @@
+package scala.scalajs.runtime
+
+import scala.annotation.StaticAnnotation
+
+class WasPublicBeforeTyper extends StaticAnnotation
```

Comment:
This should receive some Scaladoc telling not to use it. Something similar to what we have for `@RawJSType`.